maintainers/README.md

```diff
@@ -58,10 +58,35 @@ person as a reviewer, and then waiting a week for a reaction.
 
 The maintainer is welcome to come back at any time.
 
-### Tools for maintainers
+## Tools for maintainers
 
-When a pull request is made against a package, OfBorg will notify the
-appropriate maintainer(s).
+When a pull request is made against a package, nixpkgs CI will notify the
+appropriate maintainer(s) based on the package names listed in the commit titles.
+
+Maintainers can also invoke the [nixpkgs-merge-bot](https://github.com/nixos/nixpkgs-merge-bot)
+to merge pull requests targeting packages they are the maintainer of which satisfy the current
+security [constraints](https://github.com/NixOS/nixpkgs-merge-bot/blob/main/README.md#constraints).
+
+New maintainers will automatically get invited to join the [NixOS/nixpkgs-maintainers](https://github.com/orgs/NixOS/teams/nixpkgs-maintainers) GitHub team.
+By joining it you will get some triaging rights in the nixpkgs repository, like the ability to add and remove labels.
+However, the GitHub team invite is usually only sent by email and is only valid for one week!
+Should it expire please ask for a re-invite in the [GitHub org owners help desk Matrix channel](https://matrix.to/#/#org_owners:nixos.org).
+
+[zh.fail](https://zh.fail/failed/overview.html) tracks all package build failures on `master` grouped by maintainer.
+Alternatively the nixpkgs CI [hydra.nixos.org](https://hydra.nixos.org/) allows you to star any job whose status you care about.
+This results in a dashboard with an overview of all starred jobs. Example: [hydra.nixos.org/dashboard/pbsds](https://hydra.nixos.org/dashboard/pbsds@hotmail.com)
```

Comment:
Yeah, I think the way you are using it is a bit excessive, and we should build a better platform for that.

Reply:
zh.fail tracking both `master` and `release-xx.yy` could be a solution. In which repository is it developed?

Reply:
The "Next check" badge reveals https://git.helsinki.tools/janne.hess/zhf. 

(off-topic) @dasJ do you think zh.fail could track multiple jobsets like nix-review-tools-reports does? Would it also make sense to migrate the repo to the nixos github org or do you prefer to keep it where it currently is?